api/features/models.py

```diff
@@ -389,6 +389,11 @@ def to_id_priority_tuple_pairs(
     def get_audit_log_related_object_id(self, history_instance) -> int:
         return self.feature_id
 
+    def get_skip_create_audit_log(self) -> bool:
+        # Don't create audit logs when deleting feature segments using versioning
+        # v2 as we rely on the version history instead.
+        return self.environment_feature_version_id is not None
```

Comment:
nit: I would do the following
```suggestion
    @property
    def is_skip_create_audit_log(self) -> bool:
        # Don't create audit logs when deleting feature segments using versioning
        # v2 as we rely on the version history instead.
        return self.environment_feature_version_id is not None
```